updater/osv/severity.go

```diff
@@ -0,0 +1,49 @@
+// Package osv is an updater for OSV-formatted advisories.
+package osv
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/quay/claircore"
+)
+
+// check if severity (lower case) exists in the Database object if so, use it
```

Comment:
nit: we follow the go [function comments](https://tip.golang.org/doc/comment) guide of starting `ExtractSeverityFromDatabase takes an advisory.Database object ...etc`